src/makeCalculator.js

```diff
@@ -37,7 +37,45 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    add(number, thisLink) {
```

Comment:
Remove the 'thisLink' parameter from the 'add' function and use 'this' keyword inside the function body.